src/styles/calendar.scss

```diff
@@ -0,0 +1,49 @@
+$square-size: 100px;
+$day-of-week: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+$width-calendar: 728px; //$square-size * 7 + 1(gap) * 8 + 10(padding) * 2
```

Comment:
Create a file for variables and let's not hardcode width but do calculation that are commented